<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgIkAvoidance</class>
 <widget class="QDialog" name="CQDlgIkAvoidance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IK Obstacle Avoidance</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Avoidance entities</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Entity 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Entity 2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqEntity1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="qqEntity2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqDistance">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance threshold [m]</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="qqSelectEntities">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Select avoidance entities</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqClose">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqSelectEntities</tabstop>
  <tabstop>qqDistance</tabstop>
  <tabstop>qqClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
